python/cudf/cudf/core/column/decimal.py

```diff
@@ -290,6 +290,9 @@ def _with_type_metadata(
 
         return self
 
+    def _resolve_reduction_dtype(self, reduction_op: str) -> Dtype:
+        return self.dtype
```

Comment:
Looks like we don't need this override?

Reply:
If we don't have this, we end up picking up the one from `NumericalBaseColumn`.

Reply:
I think that's an indication that the definition in `NumericalBaseColumn` should be in `NumericalColumn` since that's the only case requiring special behavior. Then `Decimal*Column` will inherit the behavior of `ColumnBase` as expected.

Reply:
Oh yeah. duh